lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,33 @@
+[Gitlab Generic Packages Repository](https://docs.gitlab.com/ee/user/packages/generic_packages/) supports looking up package versions from all type of packages registry supported by Gitlab and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
```

Comment:
```suggestion
[GitLab Generic Packages Repository](https://docs.gitlab.com/ee/user/packages/generic_packages/) supports looking up package versions from all types of packages registry supported by GitLab and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
```

I think `types` should be plural?

Capitalize nouns like GitLab, GitHub, Docker, etc. 😉